requests/src/main/scala/weco/api/requests/services/SierraRequestsService.scala

```diff
@@ -276,6 +283,9 @@ class SierraRequestsService(
           identifier -> hold
         }
     } yield sourceIdentifiers.toMap
+
+  private def pickupDateHoldNote(pickupDate: LocalDate): String =
+    s"Requested for: ${DateTimeFormatter.ofPattern("yyyy-MM-dd").format(pickupDate)}"
```

Comment:
nit: I wonder if this should live in a microcopy file, a la the front-end repo?

How space-constrained are we on the note? I wonder if changing this to `Thurs 3 May 2022` would be better? Put the most relevant information first, rather than leading with the year.

Reply:
Yeah, interesting point. We need to consider how it fits on the retrievals slip - I'll leave it as is but ask Liz about viable string lengths etc. I know it was suggested to use `DD/MM/YYYY` but I would rather avoid locale-y confusion if possible, so if we can handle it I think your suggested formatting would be good.

Reply:
Sounds good, happy for you to merge this as-is.